codegen/src/main/scala/akka/grpc/gen/scaladsl/Method.scala

```diff
@@ -18,6 +18,10 @@ case class Method(
     comment: Option[String] = None)(implicit ops: DescriptorImplicits) {
   import Method._
 
+  require(
+    !ReservedWords.contains(name),
```

Comment:
All scala related fail-fast logic should be removed right?

Reply:
yes!

Reply:
I'll rebase after #1236 get merged.